projects/cn0561/builds.json

```diff
@@ -0,0 +1,12 @@
+{
+  "xilinx": {
+    "iio": {
+      "flags" : "TINYIIOD=y",
+      "hardware" : ["ad7134_fmc_zed"]
+    },
+    "uart": {
```

Comment:
also, as far as I know we have support both for zed and cora, therefore we can extend this json.

Reply:
Will do.